src/routes/_surfaces/workspace/leaderboard/MetricsTimeSeriesCharts.svelte

```diff
@@ -0,0 +1,14 @@
+<script>
+  import MetricsExploreTimeChart from "$lib/components/leaderboard/MetricsExploreTimeChart.svelte";
+  export let metricsDefId;
+</script>
+
+<div>
+  <div class="grid grid grid-flow-col">
+    <div class="big-number">
+      <h2>Place for name</h2>
+      <div />
+    </div>
+    <MetricsExploreTimeChart {metricsDefId} />
```

Comment:
@AdityaHegde this PR seems to make the `MetricsExploreTimeChart` not work as expected. Let's discuss during out 1:1.